### DISCUSSION TITLE
Automatically load a set of extensions in webapp

### DISCUSSION THREAD

**Discussion by alessandrocascino** [foxglove/community#932] [Q&A]
Hi, 
I'm trying to load a set of extensions in a self-hosted instance of foxglove-studio web app (version 1.55), but i'm not able to load these extension when the docker container is running upstream in Kubernetes by default: I tried installing them in a local instance and push the image upstream. It seems the extensions are installed only locally in my browser, but have you available any solution to have them installed by default in the image?
Thank you in advance!

**Comment by defunctzombie** [comment]
Automatically loading extensions is part of our closed-source signed-in offering and not available for the self-hosted instances.
